### PR TITLE
SSCS-7330 Add Non Prod SSCS MYA Service and Redirect

### PR DESCRIPTION
**Pull request by danhamilton**
SSCS-7330 Add Non Prod SSCS MYA Service and Redirect in advance of renaming SSCS-Cor

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by chrisdavecm — APPROVED** [review]

**Review by adusumillipraveen — CHANGES_REQUESTED** [review]
All new apps should use kustomize.

https://github.com/hmcts/cnp-flux-config/blob/master/docs/app-deployment.md

File: k8s/namespaces/sscs/sscs-mya-frontend/sscs-mya-frontend.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: sscs-mya-frontend
+  namespace: sscs
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.nodejs: glob:prod-*
```
Comment (adusumillipraveen) [inline k8s/namespaces/sscs/sscs-mya-frontend/sscs-mya-frontend.yaml:8]:
```suggestion
```

File: k8s/namespaces/sscs/sscs-mya-frontend/sscs-mya-frontend.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+spec:
+  releaseName: sscs-mya-frontend
+  rollback:
+    enable: true
+    retry: true
```
Comment (adusumillipraveen) [inline k8s/namespaces/sscs/sscs-mya-frontend/sscs-mya-frontend.yaml:13]:
```suggestion
```

File: k8s/namespaces/sscs/sscs-mya-frontend/sscs-mya-frontend.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    ref: master
+    path: stable/sscs-mya-frontend
+  values:
+    nodejs:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/sscs-mya/frontend:prod-706cd2bd
+    global:
+      tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
+      enableKeyVaults: true
```
Comment (adusumillipraveen) [inline k8s/namespaces/sscs/sscs-mya-frontend/sscs-mya-frontend.yaml:25]:
```suggestion
```

File: k8s/namespaces/sscs/sscs-mya-frontend/pertftest.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+      environment:
+        NODE_ENV: preprod
+        MYA_FEATURE_FLAG: "true"
+        ADDITIONAL_EVIDENCE_FEATURE_FLAG: "true"
+        EVIDENCE_UPLOAD_QUESTION_PAGE_OVERRIDE_ALLOWED: "true"
+        POST_BULK_SCAN: "true"
+        FT_WELSH: "true"
+        HEARING_OUTCOME_TAB_ENABLED: "true"
+    global:
+      environment: perftest
```
Comment (adusumillipraveen) [inline k8s/namespaces/sscs/sscs-mya-frontend/pertftest.yaml:18]:
```suggestion
```

File: k8s/namespaces/sscs/sscs-mya-frontend/demo.yaml [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+        NODE_ENV: preprod
+        MYA_FEATURE_FLAG: "true"
+        ADDITIONAL_EVIDENCE_FEATURE_FLAG: "true"
+        EVIDENCE_UPLOAD_QUESTION_PAGE_OVERRIDE_ALLOWED: "true"
+        POST_BULK_SCAN: "true"
+        FT_WELSH: "true"
+        TEST_PROPERTY: "true"
+        HEARING_OUTCOME_TAB_ENABLED: "true"
+    global:
+      environment: demo
```
Comment (adusumillipraveen) [inline k8s/namespaces/sscs/sscs-mya-frontend/demo.yaml:19]:
```suggestion
```

File: k8s/namespaces/sscs/sscs-mya-frontend/aat.yaml [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+      environment:
+        NODE_ENV: preprod
+        MYA_FEATURE_FLAG: "true"
+        ADDITIONAL_EVIDENCE_FEATURE_FLAG: "true"
+        EVIDENCE_UPLOAD_QUESTION_PAGE_OVERRIDE_ALLOWED: "true"
+        POST_BULK_SCAN: "true"
+        FT_WELSH: "true"
+        HEARING_OUTCOME_TAB_ENABLED: "true"
+    global:
+      environment: aat
```
Comment (adusumillipraveen) [inline k8s/namespaces/sscs/sscs-mya-frontend/aat.yaml:18]:
```suggestion
```

File: k8s/aat/common-overlay/kustomization.yaml [hunk 7]
```diff
@@ -35,6 +35,7 @@ bases:
   - camunda
   - fact
   - nfdiv
+  - sscs
```
Comment (adusumillipraveen) [inline k8s/aat/common-overlay/kustomization.yaml:38]:
```suggestion
```

File: CODEOWNERS [hunk 8]
```diff
@@ -156,6 +156,7 @@ k8s/**/common/sscs/sscs-ccd-callback-orchestrator.yaml @hmcts/sscs
 k8s/**/cluster-*/sscs/sscs-case-loader.yaml @hmcts/sscs
 k8s/**/common/sscs/sscs-tribunals-frontend.yaml @hmcts/sscs
 k8s/**/common/sscs/sscs-cor-frontend.yaml @hmcts/sscs
+k8s/**/common/sscs/sscs-mya-frontend.yaml @hmcts/sscs
```
Comment (adusumillipraveen) [inline CODEOWNERS:159]:
```suggestion
```

**Pull request closed without merging**